integration-tests/FASTN.ftd

```diff
@@ -5,4 +5,4 @@
 -- fastn.url-mappings:
 
 /ftd/* -> http+proxy://fastn.com/ftd/*
-/get-data/ -> localhost+proxy:7999/get-data/
+/test-server-data/ -> localhost+proxy:5000/get-data/
```

Comment:
What is localhost+proxy? This is wrong. It should be `http+proxy://localhost:5000/get-data/`. Also we should not allow proxy without `*`, so this line must be `/test-server-data/* -> localhost+proxy:5000/get-data/*`.